mergify_engine/github_events.py

```diff
@@ -45,9 +45,6 @@ def get_ignore_reason(event_type, data):
     elif event_type in ["push"] and not data["ref"].startswith("refs/heads/"):
         return f"push on {data['ref']}"
 
-    elif event_type == "status" and data["state"] == "pending":
```

Comment:
explanation?  :)

Reply:
We need this to put back PR in the queue in case of the PR fails and CI is rerun manually. Without this, the PR come back in the queue only later when the PR CIs succeed.

Reply:
This was initially put here to not run Mergify on each travis event, but since we batch thing, we don't really care anymore.

Reply:
Ok, so unless it's specific to this PR we could already move it in its own PR?

Reply:
We don't really need this event for the rest of mergify.